c#/RGBLedMatrix.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace rpi_rgb_led_matrix_sharp
+{
+    public class RGBLedMatrixSharp
+    {
+        [DllImport("librgbmatrix.so")]
+        public static extern IntPtr led_matrix_create(int rows, int chained, int parallel);
```

Comment:
This IntPtr looks like it is not typed, so it is easy to accidentally use wrong.
Is there a way that you can create dummy types so that the pointers that represent a Matrix and a Canvas are different ? I don't know who that is done is CS, so I let it up to you to choose the best way to do that.